gatsby-config.js

```diff
@@ -165,6 +165,72 @@ module.exports = {
         ],
       },
     },
+    {
+      resolve: "gatsby-plugin-feed-mdx",
```

Comment:
As you can see in my pr , we agreed on adding all the feed in one query, can check the comments in my pr .

Reply:
Ok..understood...so let me modify it according to your pr and then add my blog feed

Reply:
Can you check it, please?

Reply:
@subhradwip01 Yeah looks good now

Reply:
Thanks..😀